lib/Config.js

```diff
@@ -488,7 +517,6 @@ class Config {
             dataPath,
             metadataPath,
         };
-        return config;
```

Comment:
I had no idea why we did return something here. 
I removed it because of a lint return issue.